big_tests/tests/service_mongoose_system_metrics_SUITE.erl

```diff
@@ -313,6 +296,9 @@ xmpp_stanzas_counts_are_reported(Config) ->
     mongoose_helper:wait_until(fun xmpp_messages_count_is_reported/0, true),
     mongoose_helper:wait_until(fun xmpp_stanzas_counts_are_reported/0, true).
 
+config_type_is_reported(_Config) ->
+    mongoose_helper:wait_until(fun config_type_is_reported/0, true).
```

Comment:
I think it would be good to check that the correct format is reported here.

Reply:
Sure, we could add this check. Should we expect "toml" to be returned, or should we check the file format? If we were to check the file format I think I would use `mongoose_system_metrics_collector:get_config_type/0`. With such test we only test if the metric is reported, we don't check if it is collected correctly, as we get the information in the exact same way, which puts in question checking the format at all.

Reply:
Actually this is going to be a short-lived metric, I think that checking it manually for `toml` and `cfg` should be enough.

Reply:
There was no answer here, so I tested it myself and it works correctly.